src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java

```diff
@@ -99,4 +99,18 @@ public String display(Model model) {
         model.addAttribute("totalPrice", totalPrice);
         return "layout/user/header";
     }
+
+    @DeleteMapping(value = "/reset")
```

Comment:
```suggestion
    @DeleteMapping(value = "/")
```
ri dc, cần j thêm reset =)) cơ mà thôi cũng dc